src/webview/webview2/mod.rs

```diff
@@ -105,6 +111,35 @@ impl InnerWebView {
       .and_then(|path| path.to_str())
       .map(String::from);
 
+    let argument = PCWSTR::from_raw(
+      encode_wide(pl_attrs.additional_browser_args.unwrap_or_else(|| {
+        // remove "mini menu" - See https://github.com/tauri-apps/wry/issues/535
+        // and "smart screen" - See https://github.com/tauri-apps/tauri/issues/1345
+        format!(
+          "--disable-features=msWebOOUI,msPdfOOUI,msSmartScreenProtection{}{}",
+          if attributes.autoplay {
+            " --autoplay-policy=no-user-gesture-required"
+          } else {
+            ""
+          },
+          if let Some(proxy_setting) = &attributes.proxy_config {
+            match &proxy_setting {
```

Comment:
Do we need another reference here?

Reply:
Right, we don't, let me remove that real quick